docs/developer-guide.md

```diff
@@ -30,6 +29,10 @@ To build for Linux, you need to override a *PLATFORM* argument, like so:
 
     $ make release-no-tests PLATFORM=linux
 
+If `styx-test/` already exists, remove it first to avoid conflicts on dependencies:
+
+    $ rm -rf styx-test
+
 Let's unzip this file:
 
     $ unzip distribution/target/styx.zip -d styx-test
```

Comment:
This appears to be inaccurate, the file appears at `styx-<VERSION>-<OS>-<PLATFORM>.zip` instead (as described in `docs/user-guide/basic-usage.md`

Reply:
done